src/contexts/ConfigurationContext.tsx

```diff
@@ -250,7 +251,8 @@ export const ConfigContextProvider = ({ children }: Props) => {
   const [rerank, setRerank] = useState<Rerank>({
     isEnabled: false,
     numResults: 100,
-    id: 272725717
+    id: 272725717,
```

Comment:
I think the diversity reranker ID is 272725718.

Reply:
Yes this is the normal reranker. Do you think we should make the MMR the new default if rerank=true?

Reply:
Ok, how about this:
1. We have two separate options
A) rerank: true/false and rerank_num_results - this enables the current reranker (which per Amin is not really very good anyways)
B) mmr: true/false with mmr_num_results and mmr_diversity_bias specific.
that way people don't have to remember the reranker_ID and just choose A or B. If they specify both we will just turn on MMR and have a log message displayed.
wdyt?

Reply:
Sounds good!